aspnet/tutorials/nano-server/applicationHost.config

```diff
@@ -158,9 +158,9 @@
                     <binding protocol="http" bindingInformation="*:80:" />
                 </bindings>
             </site>
-            <site name="AspNet5" id="2">
+            <site name="AspNetCore" id="2">
                 <application path="/">
-                    <virtualDirectory path="/" physicalPath="C:\HelloAspNet5\wwwroot" />
+                    <virtualDirectory path="/" physicalPath="C:\HelloAspNetCore\wwwroot" />
```

Comment:
This should change to `C:\HelloAspNetCore` as web.config is no longer put into wwwroot